### ISSUE TITLE
Ich fand, dass viele Requisiten nicht verändert werden können

### ISSUE THREAD

**Issue by NightbirdChild** [Sorrow446/HLSE-Hogwarts-Legacy-Save-Editor#9]
I have obtained most of the item IDs, but most of the items cannot be modified. The system tells that the db has not been updated. What's going on?

hlse_x64.exe -i  HL-00-00.sav --item-quantities AshwinderEggs 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities Diricawlfeather 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities DittanyIeaves 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities DogbogTongue 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities Fertiliser 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities Fluxweed_Stem 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities FwooperFeather 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities GraphornHorn 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities HippogriffFeather 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities HorklumpJuice 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities JobberknollFeather 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities KneazleFur 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities KnotgrassSprig 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities LacewingFlies 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities LeadingToadstoolCaps 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities LeechJuice 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities Mallowsweet_Leavs 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities MongrelFur 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities MooncalfFur 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities Moonstone 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities NifflerFur 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities PuffskeinFur 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities ShrivelfigFruit 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities SpiderFang 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities StenchOfTheDead 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities ThestralHair 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities ToadWarts 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities TrollBogeys 100
hlse_x64.exe -i  HL-00-00.sav --item-quantities UNICORNHAIR 100
![QQ截图20230212140115](https://user-images.githubusercontent.com/48119622/218295588-303b39ee-5fa9-41db-aa58-eb6f389c3f88.jpg)

**Comment by NightbirdChild** [comment]
I have solved this problem. thank

**Comment by Sorrow446** [comment]
No problem. You can add multiple at once, by the way.

hlse_x64.exe -i HL-00-00.sav --item-quantities AshwinderEggs 100 Diricawlfeather 100

**Comment by NightbirdChild** [comment]
Yes, you said in your tutorial that you can do this, I understand. But I think it's cleaner this way, and I can confirm which one is valid and which one is invalid.

**Comment by RyanPham9x** [comment]
> I have solved this problem. thank

So how do you solve the problem? I guess you have to find the correct ID for each item, right?

**Comment by RyanPham9x** [comment]
Here's my probed Inventory with Item ID for anyone who wants to mod their save

Inventory:
Knuts, 122591
LacewingFlies, 4
Mallowsweet_Byproduct, 110
Dugbog_Pellet, 3
HorklumpJuice, 21
LeapingToadstool_Byproduct, 8
Moonstone, 110
Spider_Fang, 18
AshwinderEggs, 4
LeechJuice, 7
Fertiliser, 2
Wolf_Byproduct, 36
Dittany_Byproduct, 24
StenchOfTheDead, 80
TrollMucus, 779
Fluxweed_Byproduct, 37
Knotgrass_Byproduct, 16
Shrivelfig_Byproduct, 6
Puffskein_Byproduct, 100
Mooncalf_Byproduct, 100
Jobberknoll_Byproduct, 999
Toad_Byproduct, 99
Niffler_Byproduct, 99
Thestral_Byproduct, 101
Hippogriff_Byproduct, 101
Diricawl_Byproduct, 100
Fwooper_Byproduct, 100
Graphorn_Byproduct, 100
Kneazle_Byproduct, 100
Unicorn_Byproduct, 100
